### PR TITLE
metric nodes `depends_on`

### LINKED ISSUE #3682: Metric nodes `depends_on` will include semantic_models based on measures, but not based on filters
Somewhere in these docs:
- https://docs.getdbt.com/docs/build/measures
- https://docs.getdbt.com/docs/build/semantic-models

Let's capture that metric nodes will reflect dependencies on semantic models for their _measures_, but will not reflect dependencies based on _filters_, for the purposes of:
- dbt selection syntax
- visualizing the DAG (in dbt-docs & IDE)

### Context

(from Quigley)

There are three-ish places that metrics need to source the nodes for it's `depends_on` attribute

1. `Metric.type_params.input_metrics` for `RATIO` and `DERIVED` type metrics
2. `Metric.type_params.measure` for `SIMPLE` and `CUMULATIVE` type metrics
3.  From filters
  a. `Metric.filter`
  b. `Metric.type_params.measure.filter`
  c. `Metric.type_params.input_metrics[x].filter`

The work for 1 and 2 exist currently on dbt-labs/dbt-core#8015.

Number 3 is a bit harder though. This is because
A. The spec for filters actually isn't finalized yet. The MetricFlow team is working on support for dunders
B. Which semantic model a dimension comes from in a filter is determined at query time (because a dimension can exist on multiple semantic models and the optimal join path is calculated).

Thus this issue is just to resolve 1 & 2, whereas 3 will be pushed off and become a separate core ticket/issue

_Originally posted by @QMalcolm in https://github.com/dbt-labs/dbt-core/issues/7854#issuecomment-1622594683_

### PR DESCRIPTION
**Pull request by mirnawong1**
adding language that metric nodes include dependencies on semantic models based on measures but not filters in dag and dbt selection.

Resolves #3682

### CODE REVIEW

**Comment by mirnawong1** [comment]
hey @QMalcolm , this resolves https://github.com/dbt-labs/docs.getdbt.com/issues/3682 but would love you 👀 on this please for technical accuracy 🙏

**Comment by runleonarun** [comment]
After you review this @QMalcolm, I will jump in and use [a partial](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md) for this content so we don't have to maintain two versions of similar content. 

Nice work, Mirna!

**Review by QMalcolm — APPROVED** [review]
This looks wonderful! Thank you for doing this write-up 🚀

File: website/docs/docs/build/measures.md [hunk 1]
```diff
@@ -219,3 +219,5 @@ We can query the semi-additive metrics using the following syntax:
 mf query --metrics mrr_by_end_of_month --dimensions metric_time__month --order metric_time__month 
 mf query --metrics mrr_by_end_of_month --dimensions metric_time__week --order metric_time__week 
 ```
+
+import SetUpPages from '/snippets/_metrics-dependencies.md';
```
Comment (dbeatty10) [inline website/docs/docs/build/measures.md:223]:
```suggestion
import SetUpPages from '/snippets/_metrics-dependencies.md';

<SetUpPages /> 
```

File: website/docs/docs/build/semantic-models.md [hunk 2]
```diff
@@ -161,6 +161,24 @@ For semantic models with a measure, you must have a primary time group.
...
+- Visualization of the <Term id="dag">DAG</Term> in dbt-docs and the [integrated development environment](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) (IDE).
+
+This is because metrics need to source nodes for their `depends_on` attribute from a few different places:
+
+- `RATIO` and `DERIVED` type metrics should reference `Metric.type_params.input_metrics`.
+- `SIMPLE` <!--and `CUMULATIVE`--> type metrics should reference `Metric.type_params.measure`.
+
+For example, when you run the command `dbt list --select my_semantic_model+`, it will show you the metrics that belong to the specified semantic model.
+
+But there's a condition: Only the metrics that actually use measures or derived metrics from that semantic model will be included in the list. In other words, if a metric only uses a dimension from the semantic model in its filters, it won't be considered as part of that semantic model.
```
Comment (dbeatty10) [inline website/docs/docs/build/semantic-models.md:180]: import SetUpPages from '/snippets/_metrics-dependencies.md';

<SetUpPages />

Comment (dbeatty10) [inline website/docs/docs/build/semantic-models.md]:
```suggestion
import SetUpPages from '/snippets/_metrics-dependencies.md';

<SetUpPages />
```

**Comment by dbeatty10** [comment]
[Partial](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md) added @runleonarun !

Enabling auto-merge.

**Comment by runleonarun** [comment]
Thank you @dbeatty10!

**Pull request merged**